tests/sentry/api/endpoints/test_organization_user_teams.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import six
+
+from django.core.urlresolvers import reverse
+
+from sentry.testutils import APITestCase
+
+
+class OrganizationUserTeamsTest(APITestCase):
+    def test_simple(self):
+        foo = self.create_user("foo@example.com")
+        org = self.create_organization(owner=self.user)
+        team1 = self.create_team(organization=org)
+        team2 = self.create_team(organization=org)
+        self.create_team(organization=org)
+        self.create_member(organization=org, user=foo, teams=[team1, team2])
+        self.login_as(user=foo)
+        url = reverse(
+            "sentry-api-0-organization-user-teams", kwargs={"organization_slug": org.slug}
+        )
+        response = self.client.get(url)
+        assert response.status_code == 200
+
+        # Verify that only teams that the user is a member of, are returned
+        assert len(response.data) == 2
+        assert response.data[0]["id"] == six.text_type(team1.id)
```

Comment:
This part of the test might be unstable, since we don't have a guaranteed ordering of the teams. Probably worth just sorting the response data by `id` so that this doesn't randomly break at some point.

Reply:
oh will do